modulesync 4.0.0

bastelfreak opened the pull request:
modulesync 4.0.0

bastelfreak committed:
modulesync 4.0.0

ekohl reviewed (changes requested):
For some reason Actions didn't run.

ekohl committed:
Drop EOL EL6 support

Now that CentOS 6 is EOL and is being removed from the mirror network we
have no way to test this anymore.

[skip ci]

ekohl reviewed (approved):
Includes https://github.com/voxpupuli/puppet-telegraf/pull/159.
